lib/scooter/httpdispatchers/peboltserver_dispatcher.rb

```diff
@@ -0,0 +1,22 @@
+#require "scooter/httpdispatchers/orchestrator/v1/#{lib}"
+
+module Scooter
+  module HttpDispatchers
+    class PEBoltServerDispatcher < HttpDispatcher
+
+      #include Scooter::HttpDispatchers::PEBoltServer::V1
```

Comment:
You probably don't need this? I don't know how the versioning for the API is gonna work, so, I don't really have any strong feelings...but you can probably get rid of this comment.

Reply:
Yeah, just leftover from copy-pasting from orchestrator. Removed!